render-config.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
+DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
+VAULT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_janitor/app-sa
+VAULT_CLIENT_SERVICE_ACCOUNT_PATH=secret/dsde/terra/kernel/integration/toolsalpha/crl_janitor/client-sa
```

Comment:
Worth linking to the deployment repo where these are defined. I think we should also TODO for the same issue of wiring from terraform through to configs as the other TODO.

Reply:
Done